### DISCUSSION TITLE
after fighting with jlcpcb......

### DISCUSSION THREAD

**Discussion by dart1280** [gab-k/RP2040-Decoder#31] [General]
It took most of a day but I finally suceeded in getting to the 'order' page. Here are the compiled files I used -
[BoM.xlsx](https://github.com/user-attachments/files/18240980/BoM.xlsx)
[Placement.xlsx](https://github.com/user-attachments/files/18240982/Placement.xlsx)
[RP2040-Decoder.zip](https://github.com/user-attachments/files/18240983/RP2040-Decoder.zip)

and 

[RP2040-Decoder.elf.zip](https://github.com/user-attachments/files/18240989/RP2040-Decoder.elf.zip)

Hope that helps.

**Comment by gab-k** [comment]
Hi dart1280,

is your order already in production? If not and you can wait a little longer or are adventurous, you should perhaps wait until I merge the changes from the dev branch into the main or perhaps have the latest hardware version produced from the dev branch. The new hardware is a little better in terms of user friendliness, but I haven't validated everything yet. The changes aren't huge, but the new board has USB and is therefore a little easier to flash, etc. The only disadvantage is that there are now components on both sides of the board, but the back can be soldered fairly well by hand. If you can't cancel it anymore, it's no big deal, I write the software so that both hardware versions are still compatible. Putting together the BOM etc. is also a little easier because I already have all the part numbers from JLCPCB/LCSC in the new schematic.

Best wishes and Merry Christmas
Gabriel

**Reply by gab-k** [reply]
Oh, and also, the Fabrication toolkit for KiCad: https://github.com/bennymeg/Fabrication-Toolkit is very useful for BOM export etc. This will all be described soon in the revised documentation I am currently working on (see dev branch).

**Comment by dart1280** [comment]
Hi Gabriel, I can wait.. I looked at the $50 and decided I could wait till after Christmas.  flash via serial port huh?.. How does that work Run Project (USB) ?  So I didn't need to buy these extra RP2040's then :-)

**Reply by gab-k** [reply]
> flash via serial port huh?.. How does that work Run Project (USB) ?

https://datasheets.raspberrypi.com/pico/pico-datasheet.pdf#_programming_the_flash
